javatests/arcs/core/crdt/CrdtEntityFuzzTest.kt

```diff
@@ -0,0 +1,60 @@
+package arcs.core.crdt
+
+import arcs.core.testutil.ChooseFromList
+import arcs.core.testutil.GetInt
+import arcs.core.testutil.MapOf
+import arcs.core.testutil.SetOf
+import arcs.core.testutil.Value
+import arcs.core.testutil.runFuzzTest
+import com.google.common.truth.Truth
+import org.junit.Test
+
+class CrdtEntityFuzzTest() {
+  /**
+   * Test that a randomly generated [CrdtEntity] merged with itself produces an empty [CrdtChange].
+   */
+  @Test
+  fun crdtEntity_merge_invariant() = runFuzzTest {
```

Comment:
The test should be called something like mergeWithSelf_producesNoChanges (basically name it after the invariant but without the invariant_ prefix).